app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,19 @@
 class JobsController < ApplicationController
   def index
-    @company = Company.find(params[:company_id])
-    @jobs = @company.jobs
+    if params["sort"] == "location"
```

Comment:
this is a lot of stuff for a controller action - could we refactor this a bit? where could we move this?